thirty-five-mm/app/models/film.rb

```diff
@@ -0,0 +1,21 @@
+class Film < ActiveRecord::Base
+  belongs_to :category
+  has_many :reviews
+  has_many :ratings, as: :ratable
+  has_many :comments, as: :commentable
+
+  validates :category, :title, presence: true
+
+
+  def self.search(search)
+  	if search
+  		Film.where(title: search).to_a
```

Comment:
Looks like someone has tabs not spaces for indentation. Check your sublime settings.